code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -136,12 +136,27 @@
 	var/vine_grab_distance = 5
 	/// Whether or not this plant is ghost possessable
 	var/playable_plant = TRUE
-
+	var/withering = FALSE
 	discovery_points = 2000
 
+/mob/living/simple_animal/hostile/venus_human_trap/Initialize()
+	remove_verb(/mob/living/verb/pulled) //No pulling people into the vines
+	. = ..()
+
 /mob/living/simple_animal/hostile/venus_human_trap/Life()
 	. = ..()
 	pull_vines()
+	if(locate(/obj/structure/spacevine) in get_turf(src))//Heal if we are on vines
+		if(withering)
+			to_chat(src, "<span class='notice'> The vines nourish you, healing your wounds.</span>")
+		adjustHealth(-maxHealth*0.05)
+		withering = FALSE
+		return
+	if(!withering)
+		to_chat(src, "<span class='userdanger'>You are not being nourished by the vines and are withering away! Stay in the vines!</span>")
+	withering = TRUE
+	playsound(src.loc, 'sound/creatures/venus_trap_hurt.ogg', 50, 1)
+	adjustHealth(maxHealth*0.2)
```

Comment:
This does seem a tad steep and could be adjusted down to 0.1, but I'm not holding the PR hostage over it.

Reply:
Yea 10 seconds to death is insane, especially when you factor in that the traps will be attacked while outside of the vines. This should be something like 0.66 so that its at least 30 seconds.

Reply:
Set it to the same as the healing threshold.